RiotShareExtension/Shared/ForwardingShareItemSender.swift

```diff
@@ -16,15 +16,15 @@
 
 import Foundation
 
-let ForwardingShareItemSenderErrorDomain = "ForwardingShareItemSenderErrorDomain"
-
-enum ForwardingShareItemSenderErrorCode: Int {
-    case eventNotSentYet
-}
-
 @objcMembers
 class ForwardingShareItemSender: NSObject, ShareItemSenderProtocol {
     
+    let ErrorDomain = "ForwardingShareItemSenderErrorDomain"
```

Comment:
```suggestion
    let errorDomain = "ForwardingShareItemSenderErrorDomain"
```

Reply:
Thanks, I just realized this should be static too. You shouldn't need an instance to be able to compare error domains to it.

Reply:
It's static now, but is still `ErrorDomain` instead of `errorDomain`.

Reply:
🤦